sync_diff_inspector/diff.go

```diff
@@ -537,6 +536,25 @@ func (df *Diff) GetMatchTable(db DBConfig, schema, table string, allTables map[s
 	return tableNames, nil
 }
 
+// RemoveMatchTable removes all the matched table.
+func (df *Diff) RemoveMatchTable(table string, allTables map[string]interface{}) map[string]interface{} {
```

Comment:
I think we don't need to depend on an `map[string]interface{}` only to do this remove operation, just use slice can do the same thing.

Reply:
I prefer `map[string]struct{}`

Reply:
`map[string]struct{}` is easier to delete.